haskell/stack/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM heroku/cedar:14
+MAINTAINER Gabriel Berke-Williams <gabe@thoughtbot.com>
+
+ENV LANG C.UTF-8
+
+RUN apt-get update && \
+    apt-get install -y --force-yes wget && \
+    wget -q -O- https://s3.amazonaws.com/download.fpcomplete.com/debian/fpco.key | apt-key add - && \
+    echo 'deb http://download.fpcomplete.com/debian/jessie stable main'| tee /etc/apt/sources.list.d/fpco.list && \
```

Comment:
The Cedar image is based on Ubuntu instead of Debian, so I think this should reference an Ubuntu source?

https://github.com/thoughtbot/carnival/blob/master/Dockerfile#L11